graphql/schema/types/filters.graphql

```diff
@@ -544,7 +544,7 @@ input MultiCriterionInput {
 }
 
 input GenderCriterionInput {
-  value: GenderEnum
+  value: [GenderEnum!]
```

Comment:
This will break existing external systems, since they expect to send a single value. We generally add a new field and deprecate the old one, supporting both fields until the deprecated field is removed.